deploy/install/01-prerequisite/02-serviceaccount.yaml

```diff
@@ -72,3 +74,24 @@ subjects:
 - kind: ServiceAccount
   name: longhorn-service-account
   namespace: longhorn-system
+
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: longhorn-support-bundle
+  namespace: longhorn-system
+
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: longhorn-support-bundle
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: cluster-admin
```

Comment:
https://github.com/longhorn/longhorn/blob/master/enhancements/20221109-support-bundle-enhancement.md?plain=1#L43

Reply:
This part is a little challenging because the new support bundle is too general and designed for appliance cases. Also, if any solution uses longhorn and support bundle kit, when creating a support bundle, they will create repeated bundles, right?

Two questions here:
- Can we just download what we want? similar to the current longhorn support bundle. (this maybe needs the upstream change)
- Based on 1, can still use `simulator`?

Reply:
> * Can we just download what we want? similar to the current longhorn support bundle. (this maybe needs the upstream change)

[Here](https://github.com/harvester/harvester/blob/master/pkg/controller/master/supportbundle/manager.go#L150) are the namespaces that Harvester is targeting, including the longhorn-system namespace. So IIUC Harvesting is not going through Longhorn API to generate the support bundle, hence it shouldn't double the bundle generation.

We are only targeting the Longhorn namespace. Ref: https://github.com/rancher/support-bundle-kit/blob/master/cmd/manager.go#L47

And by default, it will also generate `kube-system` and `default` namespaces. Ref: https://github.com/rancher/support-bundle-kit/blob/master/pkg/manager/collectors/common.go#L24-L27

Here is the support bundle of the longhorn-system namespace generated with cluster-admin access.
```
> supportbundle_sample/yamls/namespaced/longhorn-system # ls
apps         coordination.k8s.io  helm.cattle.io  metrics.k8s.io     rbac.authorization.k8s.io  v1
autoscaling  discovery.k8s.io     k3s.cattle.io   networking.k8s.io  storage.k8s.io
batch        events.k8s.io        longhorn.io     policy             traefik.containo.us
```

Here is the support bundle of the longhorn-system namespace generated with longhorn-role access.
```
> supportbundle_without_cluster_admin_access/yamls/namespaced/longhorn-system # ls
apps  batch  coordination.k8s.io  longhorn.io  metrics.k8s.io  policy  v1
```

> * Based on 1, can still use `simulator`?
The simulator can simulate both support bundles. However, the one without cluster-admin access will be missing some resources.

Simulate with cluster-admin access
```
NAMESPACE         NAME                                                                  READY   STATUS              RESTARTS   AGE
default           pod/longhorn-test                                                     2/2     Running             0          25h
default           pod/volume-test                                                       1/1     Running             0          117m
kube-system       pod/coredns-7796b77cd4-zd2bs                                          1/1     Running             0          12d
kube-system       pod/helm-install-traefik-crd-fmwc6                                    0/1     Completed           0          12d
kube-system       pod/helm-install-traefik-wm5wv                                        0/1     Completed           1          12d
kube-system       pod/local-path-provisioner-84bb864455-rnfv2                           1/1     Running             0          12d
kube-system       pod/metrics-server-ff9dbcb6c-tp87g                                    1/1     Running             0          12d
kube-system       pod/svclb-traefik-76gv5                                               2/2     Running             0          12d
kube-system       pod/svclb-traefik-7cxjd                                               2/2     Running             0          12d
kube-system       pod/svclb-traefik-fg8wq                                               2/2     Running             0          12d
kube-system       pod/traefik-d497b4cb6-pxlnk                                           1/1     Running             0          12d
longhorn-system   pod/csi-attacher-84b96d64c8-5ft7s                                     1/1     Running             1          25h
longhorn-system   pod/csi-attacher-84b96d64c8-7p5fp                                     1/1     Running             1          25h
longhorn-system   pod/csi-attacher-84b96d64c8-tvs5h                                     1/1     Running             1          25h
longhorn-system   pod/csi-provisioner-6ccbfbf86f-25c6c                                  1/1     Running             1          25h
longhorn-system   pod/csi-provisioner-6ccbfbf86f-b976c                                  1/1     Running             1          25h
longhorn-system   pod/csi-provisioner-6ccbfbf86f-pk7r9                                  1/1     Running             1          25h
longhorn-system   pod/csi-resizer-58c959486d-6k6zj                                      1/1     Running             1          25h
longhorn-system   pod/csi-resizer-58c959486d-jxfff                                      1/1     Running             1          25h
longhorn-system   pod/csi-resizer-58c959486d-t5djd                                      1/1     Running             1          25h
longhorn-system   pod/csi-snapshotter-7f744fbb67-dfpfs                                  1/1     Running             0          25h
longhorn-system   pod/csi-snapshotter-7f744fbb67-lz7r4                                  1/1     Running             0          25h
longhorn-system   pod/csi-snapshotter-7f744fbb67-vsr64                                  1/1     Running             0          25h
longhorn-system   pod/demo-0-restore-6xfmf                                              0/1     Completed           0          4d19h
longhorn-system   pod/engine-image-ei-70a94952-2hfmq                                    1/1     Running             0          5d23h
longhorn-system   pod/engine-image-ei-70a94952-rzvxc                                    1/1     Running             0          5d23h
longhorn-system   pod/engine-image-ei-70a94952-vq7nf                                    1/1     Running             0          5d23h
longhorn-system   pod/instance-manager-e-6e643731                                       1/1     Running             0          3d20h
longhorn-system   pod/instance-manager-e-ee0ec3ea                                       1/1     Running             0          3d20h
longhorn-system   pod/instance-manager-e-f58ea986                                       1/1     Running             0          3d20h
longhorn-system   pod/instance-manager-r-01a28715                                       1/1     Running             0          3d20h
longhorn-system   pod/instance-manager-r-36665b32                                       1/1     Running             0          3d20h
longhorn-system   pod/instance-manager-r-be3a2a8d                                       1/1     Running             0          3d20h
longhorn-system   pod/longhorn-admission-webhook-59ff547b8b-4szh5                       1/1     Running             0          22h
longhorn-system   pod/longhorn-admission-webhook-59ff547b8b-9jl49                       1/1     Running             0          22h
longhorn-system   pod/longhorn-conversion-webhook-78d89d8667-4xczr                      1/1     Running             0          22h
longhorn-system   pod/longhorn-conversion-webhook-78d89d8667-knf6q                      1/1     Running             0          22h
longhorn-system   pod/longhorn-csi-plugin-26f2b                                         2/2     Running             0          25h
longhorn-system   pod/longhorn-csi-plugin-mq4zh                                         2/2     Running             0          25h
longhorn-system   pod/longhorn-csi-plugin-zdrcf                                         2/2     Running             0          25h
longhorn-system   pod/longhorn-driver-deployer-77f5f7d955-fcq49                         1/1     Running             0          22h
longhorn-system   pod/longhorn-manager-9q87w                                            1/1     Running             0          22h
longhorn-system   pod/longhorn-manager-j4mbh                                            1/1     Running             0          22h
longhorn-system   pod/longhorn-manager-l6h2c                                            1/1     Running             0          22h
longhorn-system   pod/longhorn-recovery-backend-6764d865c6-8d4ck                        1/1     Running             0          22h
longhorn-system   pod/longhorn-recovery-backend-6764d865c6-vmt2b                        1/1     Running             0          22h
longhorn-system   pod/longhorn-support-bundle-manager-support-bundle-2022-11-22tfvswz   0/1     ContainerCreating   0          37m
longhorn-system   pod/longhorn-ui-594d98bfdc-6qdl7                                      1/1     Running             0          4d21h
longhorn-system   pod/supportbundle-manager-sample-768d88d455-trmtv                     1/1     Running             0          7d16h

NAMESPACE         NAME                                   TYPE           CLUSTER-IP      EXTERNAL-IP                        PORT(S)                      AGE
default           service/kubernetes                     ClusterIP      10.43.0.1       <none>                             443/TCP                      <unknown>
kube-system       service/kube-dns                       ClusterIP      10.43.0.10      <none>                             53/UDP,53/TCP,9153/TCP       12d
kube-system       service/metrics-server                 ClusterIP      10.43.165.104   <none>                             443/TCP                      12d
kube-system       service/traefik                        LoadBalancer   10.43.69.102    10.0.1.102,10.0.1.105,10.0.1.113   80:30821/TCP,443:31679/TCP   12d
longhorn-system   service/csi-attacher                   ClusterIP      10.43.31.113    <none>                             12345/TCP                    25h
longhorn-system   service/csi-provisioner                ClusterIP      10.43.19.143    <none>                             12345/TCP                    25h
longhorn-system   service/csi-resizer                    ClusterIP      10.43.27.77     <none>                             12345/TCP                    25h
longhorn-system   service/csi-snapshotter                ClusterIP      10.43.117.1     <none>                             12345/TCP                    25h
longhorn-system   service/longhorn-admission-webhook     ClusterIP      10.43.57.63     <none>                             9443/TCP                     12d
longhorn-system   service/longhorn-backend               ClusterIP      10.43.180.125   <none>                             9500/TCP                     12d
longhorn-system   service/longhorn-conversion-webhook    ClusterIP      10.43.93.167    <none>                             9443/TCP                     12d
longhorn-system   service/longhorn-engine-manager        ClusterIP      None            <none>                             <none>                       12d
longhorn-system   service/longhorn-frontend              NodePort       10.43.105.21    <none>                             80:30111/TCP                 12d
longhorn-system   service/longhorn-recovery-backend      ClusterIP      10.43.147.126   <none>                             9600/TCP                     12d
longhorn-system   service/longhorn-replica-manager       ClusterIP      None            <none>                             <none>                       12d
longhorn-system   service/supportbundle-manager-sample   NodePort       10.43.59.25     <none>                             8080:30116/TCP               7d16h

NAMESPACE         NAME                                      DESIRED   CURRENT   READY   UP-TO-DATE   AVAILABLE   NODE SELECTOR   AGE
kube-system       daemonset.apps/svclb-traefik              3         3         3       3            3           <none>          12d
longhorn-system   daemonset.apps/engine-image-ei-70a94952   3         3         3       3            3           <none>          5d23h
longhorn-system   daemonset.apps/longhorn-csi-plugin        3         3         3       3            3           <none>          25h
longhorn-system   daemonset.apps/longhorn-manager           3         3         3       3            3           <none>          12d

NAMESPACE         NAME                                                                                  READY   UP-TO-DATE   AVAILABLE   AGE
kube-system       deployment.apps/coredns                                                               1/1     1            1           12d
kube-system       deployment.apps/local-path-provisioner                                                1/1     1            1           12d
kube-system       deployment.apps/metrics-server                                                        1/1     1            1           12d
kube-system       deployment.apps/traefik                                                               1/1     1            1           12d
longhorn-system   deployment.apps/csi-attacher                                                          3/3     3            3           25h
longhorn-system   deployment.apps/csi-provisioner                                                       3/3     3            3           25h
longhorn-system   deployment.apps/csi-resizer                                                           3/3     3            3           25h
longhorn-system   deployment.apps/csi-snapshotter                                                       3/3     3            3           25h
longhorn-system   deployment.apps/longhorn-admission-webhook                                            2/2     2            2           12d
longhorn-system   deployment.apps/longhorn-conversion-webhook                                           2/2     2            2           12d
longhorn-system   deployment.apps/longhorn-driver-deployer                                              1/1     1            1           12d
longhorn-system   deployment.apps/longhorn-recovery-backend                                             2/2     2            2           12d
longhorn-system   deployment.apps/longhorn-support-bundle-manager-support-bundle-2022-11-22t02-37-32z   0/1     1            0           37m
longhorn-system   deployment.apps/longhorn-ui                                                           1/1     1            1           12d
longhorn-system   deployment.apps/supportbundle-manager-sample                                          1/1     1            1           7d16h

NAMESPACE         NAME                                                                                             DESIRED   CURRENT   READY   AGE
kube-system       replicaset.apps/coredns-7796b77cd4                                                               1         1         1       12d
kube-system       replicaset.apps/local-path-provisioner-84bb864455                                                1         1         1       12d
kube-system       replicaset.apps/metrics-server-ff9dbcb6c                                                         1         1         1       12d
kube-system       replicaset.apps/traefik-d497b4cb6                                                                1         1         1       12d
longhorn-system   replicaset.apps/csi-attacher-84b96d64c8                                                          3         3         3       25h
longhorn-system   replicaset.apps/csi-provisioner-6ccbfbf86f                                                       3         3         3       25h
longhorn-system   replicaset.apps/csi-resizer-58c959486d                                                           3         3         3       25h
longhorn-system   replicaset.apps/csi-snapshotter-7f744fbb67                                                       3         3         3       25h
longhorn-system   replicaset.apps/longhorn-admission-webhook-54b77d6c5d                                            0         0         0       23h
longhorn-system   replicaset.apps/longhorn-admission-webhook-57cb795894                                            0         0         0       3d22h
longhorn-system   replicaset.apps/longhorn-admission-webhook-5868cd7d46                                            0         0         0       23h
longhorn-system   replicaset.apps/longhorn-admission-webhook-59ff547b8b                                            2         2         2       22h
longhorn-system   replicaset.apps/longhorn-admission-webhook-5d6cdc7dc8                                            0         0         0       25h
longhorn-system   replicaset.apps/longhorn-admission-webhook-5d885dddd6                                            0         0         0       3d19h
longhorn-system   replicaset.apps/longhorn-admission-webhook-5d9bdd9cc8                                            0         0         0       3d18h
longhorn-system   replicaset.apps/longhorn-admission-webhook-766589678b                                            0         0         0       3d19h
longhorn-system   replicaset.apps/longhorn-admission-webhook-84dbfcf978                                            0         0         0       3d22h
longhorn-system   replicaset.apps/longhorn-admission-webhook-98d8d996b                                             0         0         0       3d18h
longhorn-system   replicaset.apps/longhorn-admission-webhook-99d569666                                             0         0         0       3d20h
longhorn-system   replicaset.apps/longhorn-conversion-webhook-54ccd6fb84                                           0         0         0       3d22h
longhorn-system   replicaset.apps/longhorn-conversion-webhook-54cdfd7bff                                           0         0         0       3d18h
longhorn-system   replicaset.apps/longhorn-conversion-webhook-56f54cb6f8                                           0         0         0       23h
longhorn-system   replicaset.apps/longhorn-conversion-webhook-57d867c5b                                            0         0         0       3d19h
longhorn-system   replicaset.apps/longhorn-conversion-webhook-5948df5f7d                                           0         0         0       3d20h
longhorn-system   replicaset.apps/longhorn-conversion-webhook-596f5c5ccc                                           0         0         0       3d19h
longhorn-system   replicaset.apps/longhorn-conversion-webhook-5b4cf99fcf                                           0         0         0       3d18h
longhorn-system   replicaset.apps/longhorn-conversion-webhook-5d954b77c6                                           0         0         0       25h
longhorn-system   replicaset.apps/longhorn-conversion-webhook-78d89d8667                                           2         2         2       22h
longhorn-system   replicaset.apps/longhorn-conversion-webhook-795cb9c8c5                                           0         0         0       3d22h
longhorn-system   replicaset.apps/longhorn-conversion-webhook-85bff777fc                                           0         0         0       23h
longhorn-system   replicaset.apps/longhorn-driver-deployer-55c76fc578                                              0         0         0       3d20h
longhorn-system   replicaset.apps/longhorn-driver-deployer-5cffc98769                                              0         0         0       3d19h
longhorn-system   replicaset.apps/longhorn-driver-deployer-677764c9f8                                              0         0         0       23h
longhorn-system   replicaset.apps/longhorn-driver-deployer-7684684f76                                              0         0         0       3d22h
longhorn-system   replicaset.apps/longhorn-driver-deployer-77f5f7d955                                              1         1         1       22h
longhorn-system   replicaset.apps/longhorn-driver-deployer-7ddf7cc744                                              0         0         0       25h
longhorn-system   replicaset.apps/longhorn-driver-deployer-85f5f87d9f                                              0         0         0       3d22h
longhorn-system   replicaset.apps/longhorn-driver-deployer-8766bbbff                                               0         0         0       3d18h
longhorn-system   replicaset.apps/longhorn-driver-deployer-87f5fb4f9                                               0         0         0       3d18h
longhorn-system   replicaset.apps/longhorn-driver-deployer-f8d75cc89                                               0         0         0       23h
longhorn-system   replicaset.apps/longhorn-driver-deployer-fdb658d44                                               0         0         0       3d19h
longhorn-system   replicaset.apps/longhorn-recovery-backend-56fd9786b5                                             0         0         0       25h
longhorn-system   replicaset.apps/longhorn-recovery-backend-57c5df88f4                                             0         0         0       3d19h
longhorn-system   replicaset.apps/longhorn-recovery-backend-589f845b8d                                             0         0         0       3d18h
longhorn-system   replicaset.apps/longhorn-recovery-backend-598ddd86b5                                             0         0         0       3d19h
longhorn-system   replicaset.apps/longhorn-recovery-backend-5f754ffc44                                             0         0         0       3d22h
longhorn-system   replicaset.apps/longhorn-recovery-backend-65f47585b7                                             0         0         0       23h
longhorn-system   replicaset.apps/longhorn-recovery-backend-6764d865c6                                             2         2         2       22h
longhorn-system   replicaset.apps/longhorn-recovery-backend-84cc489ff4                                             0         0         0       23h
longhorn-system   replicaset.apps/longhorn-recovery-backend-8576db48c5                                             0         0         0       3d20h
longhorn-system   replicaset.apps/longhorn-recovery-backend-864c4db49c                                             0         0         0       3d22h
longhorn-system   replicaset.apps/longhorn-recovery-backend-c6669c468                                              0         0         0       3d18h
longhorn-system   replicaset.apps/longhorn-support-bundle-manager-support-bundle-2022-11-22t02-37-32z-7649b9f47c   1         1         0       37m
longhorn-system   replicaset.apps/longhorn-ui-594d98bfdc                                                           1         1         1       12d
longhorn-system   replicaset.apps/longhorn-ui-7c66b4d4f8                                                           0         0         0       4d21h
longhorn-system   replicaset.apps/supportbundle-manager-sample-768d88d455                                          1         1         1       7d16h

NAMESPACE         NAME                                 COMPLETIONS   DURATION   AGE
kube-system       job.batch/helm-install-traefik       1/1           44s        12d
kube-system       job.batch/helm-install-traefik-crd   1/1           42s        12d
longhorn-system   job.batch/demo-0-restore             1/1           30s        4d19h
```

Simulate with longhorn-role access.
```
NAMESPACE         NAME                                   TYPE           CLUSTER-IP      EXTERNAL-IP                        PORT(S)                      AGE
default           service/kubernetes                     ClusterIP      10.43.0.1       <none>                             443/TCP                      <unknown>
kube-system       service/kube-dns                       ClusterIP      10.43.0.10      <none>                             53/UDP,53/TCP,9153/TCP       12d
kube-system       service/metrics-server                 ClusterIP      10.43.165.104   <none>                             443/TCP                      12d
kube-system       service/traefik                        LoadBalancer   10.43.69.102    10.0.1.102,10.0.1.105,10.0.1.113   80:30821/TCP,443:31679/TCP   12d
longhorn-system   service/csi-attacher                   ClusterIP      10.43.31.113    <none>                             12345/TCP                    23h
longhorn-system   service/csi-provisioner                ClusterIP      10.43.19.143    <none>                             12345/TCP                    23h
longhorn-system   service/csi-resizer                    ClusterIP      10.43.27.77     <none>                             12345/TCP                    23h
longhorn-system   service/csi-snapshotter                ClusterIP      10.43.117.1     <none>                             12345/TCP                    23h
longhorn-system   service/longhorn-admission-webhook     ClusterIP      10.43.57.63     <none>                             9443/TCP                     12d
longhorn-system   service/longhorn-backend               ClusterIP      10.43.180.125   <none>                             9500/TCP                     12d
longhorn-system   service/longhorn-conversion-webhook    ClusterIP      10.43.93.167    <none>                             9443/TCP                     12d
longhorn-system   service/longhorn-engine-manager        ClusterIP      None            <none>                             <none>                       12d
longhorn-system   service/longhorn-frontend              NodePort       10.43.105.21    <none>                             80:30111/TCP                 12d
longhorn-system   service/longhorn-recovery-backend      ClusterIP      10.43.147.126   <none>                             9600/TCP                     12d
longhorn-system   service/longhorn-replica-manager       ClusterIP      None            <none>                             <none>                       12d
longhorn-system   service/supportbundle-manager-sample   NodePort       10.43.59.25     <none>                             8080:30116/TCP               7d15h

NAMESPACE         NAME                                      DESIRED   CURRENT   READY   UP-TO-DATE   AVAILABLE   NODE SELECTOR   AGE
kube-system       daemonset.apps/svclb-traefik              3         3         3       3            3           <none>          12d
longhorn-system   daemonset.apps/engine-image-ei-70a94952   3         3         3       3            3           <none>          5d21h
longhorn-system   daemonset.apps/longhorn-csi-plugin        3         3         3       3            3           <none>          23h
longhorn-system   daemonset.apps/longhorn-manager           3         3         3       3            3           <none>          12d

NAMESPACE         NAME                                                                                  READY   UP-TO-DATE   AVAILABLE   AGE
kube-system       deployment.apps/coredns                                                               1/1     1            1           12d
kube-system       deployment.apps/local-path-provisioner                                                1/1     1            1           12d
kube-system       deployment.apps/metrics-server                                                        1/1     1            1           12d
kube-system       deployment.apps/traefik                                                               1/1     1            1           12d
longhorn-system   deployment.apps/csi-attacher                                                          3/3     3            3           23h
longhorn-system   deployment.apps/csi-provisioner                                                       3/3     3            3           23h
longhorn-system   deployment.apps/csi-resizer                                                           3/3     3            3           23h
longhorn-system   deployment.apps/csi-snapshotter                                                       3/3     3            3           23h
longhorn-system   deployment.apps/longhorn-admission-webhook                                            2/2     2            2           12d
longhorn-system   deployment.apps/longhorn-conversion-webhook                                           2/2     2            2           12d
longhorn-system   deployment.apps/longhorn-driver-deployer                                              1/1     1            1           12d
longhorn-system   deployment.apps/longhorn-recovery-backend                                             2/2     2            2           12d
longhorn-system   deployment.apps/longhorn-support-bundle-manager-support-bundle-2022-11-22t01-18-38z   1/1     1            1           28m
longhorn-system   deployment.apps/longhorn-ui                                                           1/1     1            1           12d
longhorn-system   deployment.apps/supportbundle-manager-sample                                          1/1     1            1           7d15h

NAMESPACE         NAME                                 COMPLETIONS   DURATION   AGE
kube-system       job.batch/helm-install-traefik       1/1           44s        12d
kube-system       job.batch/helm-install-traefik-crd   1/1           42s        12d
longhorn-system   job.batch/demo-0-restore             1/1           30s        4d18h
```

Reply:
Note: moved to https://github.com/longhorn/longhorn/pull/4915